### DISCUSSION TITLE
I need help!!!!

### DISCUSSION THREAD

**Discussion by nxvvvv** [upptime/upptime#218] [Q&A] [answered]
Why my [status page](https://navaneethkm004.github.io/uptime/) is looking like that but the [demo](https://demo.upptime.js.org) is like that?

**Comment by nxvvvv** [comment]
![image](https://user-images.githubusercontent.com/34748927/104808303-4c204580-580b-11eb-9a14-32f5aa29b6db.png)

**Comment by AnandChowdhary** [comment, marked as answer]
You didn't select gh-pages as the deployment branch: https://upptime.js.org/docs/get-started#enable-publishing

Please use discussions for Q&A.

**Reply by nxvvvv** [reply]
Bro but look at this 
![image](https://user-images.githubusercontent.com/34748927/104809192-69581280-5811-11eb-8e33-aeed3e5a8453.png)

**Reply by nxvvvv** [reply]
Ohh now I did everything again and chose gh-pages. But the same result. 
![image](https://user-images.githubusercontent.com/34748927/104809378-d324ec00-5812-11eb-8a32-e8b39157a786.png)
![image](https://user-images.githubusercontent.com/34748927/104809386-e46df880-5812-11eb-90d9-0ea23527cacd.png)

**Reply by nxvvvv** [reply]
Now its worse bro

![image](https://user-images.githubusercontent.com/34748927/104809415-0ff0e300-5813-11eb-8df1-884cc67a4470.png)

**Reply by AnandChowdhary** [reply]
Please follow the get started guide: https://upptime.js.org/docs/get-started.

You have to set your configuration right with the `baseUrl`: https://upptime.js.org/docs/get-started#update-configuration.

**Reply by nxvvvv** [reply]
@AnandChowdhary Thank you very much Bro!!! It worked!!

**Reply by nxvvvv** [reply]
Also, how to change the theme color from light to dark. Is there any settings or should i do it on my own?

**Reply by nxvvvv** [reply]
@AnandChowdhary
